### PR TITLE
Update paste-ssh-public-key.md

### PR DESCRIPTION
**Pull request by romw314**
### Why:
The path is different than the path in the code block!

Closes: 
**`NOTHING`**

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by romw314** [comment]
What's `Repo Freeze Check / Prevent merging during deployment freezes (pull_request_target)`?

File: data/reusables/gpg/paste-ssh-public-key.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-1. To set your SSH signing key in Git, paste the text below, substituting **/PATH/TO/KEY.PUB** with the path to the public key you'd like to use.
+1. To set your SSH signing key in Git, paste the text below, substituting **/PATH/TO/.SSH/KEY.PUB** with the path to the public key you'd like to use.
 
    ```bash
    $ git config --global user.signingkey /PATH/TO/.SSH/KEY.PUB
```
Comment (jhosman) [inline data/reusables/gpg/paste-ssh-public-key.md:4]:
```suggestion
   ```bash
   $ git config --global user.signingkey /PATH/TO/.SSH/KEY.PUB
   ```
```

**Review by jhosman — COMMENTED** [review]
Thanks for noticing this and opening a PR with the fix! ✨  💖 ✨ 

While in here, I noticed an unrelated error we can also fix - It looks like the bash code block is missing the closing backtick characters, which is causing the code block to render with extra space. I'll get that change incorporated as well, and then we'll get ready to merge the PR for you 🚀 (Note that it will be a couple of days before we merge the PR, because the repo is temporarily frozen right now.)

Thanks again for helping to improve GitHub's docs!

**Review by jhosman — DISMISSED** [review]

**Review by jhosman — APPROVED** [review]

**Comment by romw314** [comment]
Hi, @jhosman, you can now merge this PR.

**Pull request merged**